live-ingester/src/main/resources/application.yml

```diff
@@ -34,9 +34,6 @@ alfresco:
     shared-file-store:
       host: http://localhost
       port: 8099
-    mime-type:
-      mapping:
-        '[*]': "application/pdf"
```

Comment:
I still think it's worth providing some default settings here. Maybe something similar to what you've used in the e2es above? Possibly exactly what you've done for images and then a curated list of mime-types that will convert to pdf?

Reply:
Or alternatively we could make the default no-config behaviour more complex, but I think just including what we think customers will want here is best.

Reply:
Honestly, I had several ideas on how to tackle that. 
I ended up with being sort of backwards compatible - when there is no config in the properties - the target mime-type is always `application/pdf` (we used to do that until this change came in).
Any other configs reflected in properties can break that "compatibility" and I thought they'd have to be consciously set up.
I am happy to put E2Es alike config here
``` yml
    mime-type:
      mapping:
        '[image/png]': "image/png"
        '[image/bmp]': "image/png"
        '[image/gif]': "image/png"
        '[image/raw]': "image/png"
        '[image/*]': "image/jpeg"
        '[*]': "application/pdf"
```
Shall I do that?

Reply:
I think that's a reasonable first version. I did wonder if we could try to find a list of all mimetypes that can be converted to pdf by the default ATS set up, and then leave everything else so that it won't attempt conversion. I'm happy with this suggestion though.

Reply:
I've found [this](https://docs.alfresco.com/content-services/latest/admin/transformations/) - quite a long list of possible transformations. I'll have a look at that and figure out whether it can be compressed to a reasonable list.

Reply:
Nice find. Looks like we could get a reasonably improvement by defaulting to all of `[application/*]` and `[text/*]` converted to pdf.

Reply:
Yep, will do that - seems to be a reasonable default.